src/scales/bookoo.cpp

```diff
@@ -0,0 +1,250 @@
+#include "bookoo.h"
+#include "remote_scales_plugin_registry.h"
+
+/*
+Handle protocol according to the spec found at
+https://github.com/BooKooCode/OpenSource/blob/main/bookoo_mini_scale/protocols.md
+*/
+
+enum class BookooHeader : uint8_t {
```

Comment:
looks redundant?

Reply:
Yup spotted that right, i left hem there as was thinking they are needed originally but ended up unnecessary and i just forgot to slay lol